src/Helper/Normalizer.php

```diff
@@ -106,13 +105,22 @@ public static function renderLabel(
     ): string {
         $html = '';
 
+        /** @psalm-var non-empty-string $tagName */
```

Comment:
Remove this annotation and add `@psalm-return non-empty-string` to `iconTagName()` method

Reply:
done.